src/tests/SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests/Services/TestPaymentsProcessFunctions.cs

```diff
@@ -1,37 +1,37 @@
-﻿using Microsoft.Extensions.Configuration;
-using Newtonsoft.Json;
+﻿using Newtonsoft.Json;
 using Polly;
+using SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests.AzureFunctions;
 using System;
-using System.Diagnostics;
-using System.IO;
+using System.Collections.Generic;
+using System.Linq;
 using System.Net.Http;
 using System.Text;
-using System.Threading;
 using System.Threading.Tasks;
-using System.Web;
 
 namespace SFA.DAS.EmployerIncentives.Functions.PaymentsProcess.AcceptanceTests.Services
 {
     public class TestPaymentsProcessFunctions : IDisposable
     {
-        private const string TestConfigFile = "local.settings.json";
-        private readonly TestContext _testContext;
         private const int Port = 7007;
-        private readonly Settings _settings = new Settings();
-        private readonly HttpClient _client;
+        private const int StartupTimeoutInSeconds = 30;
+        private const int OrchestratorRunTimeoutInSeconds = 10;
+
         private bool _isDisposed;
-        private Process _functionHostProcess;
+        private readonly HttpClient _client;
+        private readonly FunctionsController _functionsController;
+        private readonly TestPaymentsProcessFunctionsConfigurator _configurator;
 
-        public TestPaymentsProcessFunctions(TestContext testContext)
+        public TestPaymentsProcessFunctions(string databaseConnectionString, string learnerMatchApiBaseUrl)
```

Comment:
I think you should still pass in just the TestContext and get the information required from it.  Then you are covered for future changes or if other scenarios need extra information.